File: Basic/find average/find average.js
array=[1,2,3,"Jess", -1, 55]
avg=0
arr_length=0

for(var i=0; i<array.length; i++){
	if(typeof array[i]== "number"){
		avg+=array[i]
		arr_length++
	}
}

avg=avg/arr_length
console.log(avg)

File: Basic/find average/find average.py
array=[1,2,3,"Jess", -1, 55]
sum=0
end=len(array)
arr_length=0

for i in range(0, end):
	if type(array[i]) == int:
		arr_length+=1
		sum+= array[i]

sum=sum/arr_length
print(sum)